harvard-lts/drs-translation-service
# Batch Builder Service for ETDs

**ETD Batch Builder Service implementation (specifically for doc/text)**
* * *

**JIRA Issue**: https://at-harvard.atlassian.net/browse/ETD-279

# How should this be tested?

Run pytests (see [README](https://github.com/harvard-lts/drs-translation-service#testing))
Run the integration test: https://ltsds-cloud-dev-1.lib.harvard.edu:10610/etd_test

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? Yes
- integration tests? Yes (in etd-integration-tests)
